### DISCUSSION TITLE
Home redesign

### DISCUSSION THREAD

**Discussion by fmerian** [dubinc/dub#359] [Ideas]
spotted on X -- [NomanDesigns](https://twitter.com/Nomandsign) made a refresh of [Dub's homepage](https://twitter.com/Nomandsign/status/1696896179336823053?s=20).

would this be something to consider?

**Comment by steven-tey** [comment]
Love the concept, but probably not going to do a redesign any time soon (need to focus on API and some feature pages first :D)

Thanks anyway!
